onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1375,6 +1375,21 @@ ONNX_MS_OPERATOR_SET_SCHEMA(Sampling, 1,
                                   GreedySearchShapeInference(ctx);
                                 }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(MoEBlock, 1,
+                            OpSchema()
+                                .SetDoc("Mixture of experts.")
+                                .Attr("activation_type", "Activation function to use", AttributeProto::STRING, std::string("relu"))
+                                .Attr("k", "Number of top experts to select from expert pool", AttributeProto::INT, static_cast<int64_t>(1))
+                                .Input(0, "input", "2D input tensor with shape (num_rows, hidden_size)", "T")
+                                .Input(1, "gated_output", "2D input tensor with shape (num_rows, num_experts)", "T")
```

Comment:
Is it proper to rename the input as router_probs?

Reply:
sounds good